unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Scripts/ControlsModule/SettingsControlModel.cs

```diff
@@ -20,21 +21,28 @@ public class SettingsControlGroup
         public SettingsControlList controls;
     }
 
-    [CreateAssetMenu(menuName = "Settings/Configuration/Controls/Default Control", fileName = "DefaultControlConfiguration")]
     public class SettingsControlModel : ScriptableObject
     {
+        [Header("CONTROL CONFIGURATION")]
         public string title;
         public SettingsControlView controlPrefab;
         public SettingsControlController controlController;
+        public List<BooleanVariable> flagsThatDisablesMe;
+        public List<BooleanVariable> flagsThatDeactivatesMe;
```

Comment:
minor correction XD: not necessarily for this PR, but we should remove one "S" in both like `flagsThatDisableMe` and `flagsThatDeactivateMe`

Reply:
Oh! Good catch, thanks Pravus! I will fix it 👍

Reply:
Done in the this PR: https://github.com/decentraland/explorer/pull/1784